apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -229,7 +235,11 @@ export default function RubricsContainer({
       if (Array.isArray(value)) {
         newObj[snakeCase(key)] = value.map(item => transformObjectKeys(item));
       } else {
-        newObj[snakeCase(key)] = value;
+        if (key === '_destroy') {
```

Comment:
I needed to add this so that it wouldn't try to turn this into snakeCase.  It was already formatted correctly with `_destroy`

Reply:
Makes sense!

Another way to write it that's arguably WORSE lol is simply:

```
newObj[key === '_destroy' ? key : snakeCase(key)] = value
```

But somebody might justifiably come along and say "burn that down, please" because they hate the ternary. They think it is ugly. I love the ternary for cases like this! :)

Then there is something overly clever like:

```
newObj[snakeCase('x' + key).substr(1)] = value
```

Much more obviously deserving of fire.

Definitely good to just be very descriptive... use as many lines as it needs to be clear. So your way is the best way. Looks great!